app/javascript/components/tree-view/reducers/basics.js

```diff
@@ -0,0 +1,24 @@
+import Tree from '@manageiq/react-ui-components/dist/wooden-tree';
```

Comment:
The name is confusing, shouldn't it be `basicStore`?

Reply:
It is in the `reducers` folder... so it contains the basic reducers? :)

Reply:
Well, but you're importing the exported thing from it as `basicStore`?

Reply:
It sounds better than `basicReducers` or `reducersBasics`? This is why the default export is good :D.

Reply:
You should probably name the file as the default exported thing, especially if you're not exporting anyhing else from there.